consensus/checks.go

```diff
@@ -174,18 +176,28 @@ func (consensus *Consensus) onViewChangeSanityCheck(recvMsg *FBFTMessage) bool {
 	}
 	if consensus.blockNum < recvMsg.BlockNum {
 		consensus.getLogger().Warn().
-			Msg("[onViewChange] New Leader Has Lower Blocknum")
+			Msg("[onViewChangeSanityCheck] MsgBlockNum is different from my BlockNumber")
 		return false
 	}
 	if consensus.IsViewChangingMode() &&
-		consensus.GetViewChangingID() > recvMsg.ViewID {
+		consensus.GetCurBlockViewID() > recvMsg.ViewID {
```

Comment:
So this is intentionally changed so we are more lenient on the viewID received?

Reply:
The true lower bound is curBlockViewID, not the viewchangingID.

Reply:
cool. just double check that this change is intentional.